tests/conftest.py

```
@@ -1,14 +1,35 @@
+    each async test. httpx ``AsyncClient`` instances held by providers are finalized
+    afterwards and schedule ``aclose()`` on the already-closed loop, which asyncio reports
+    through its logger as ``RuntimeError: Event loop is closed``. These are teardown
+    artifacts, not real failures; every other asyncio record passes through untouched.
+    """
+
+    @override
+    def filter(self, record: logging.LogRecord) -> bool:
+        return "Event loop is closed" not in record.getMessage()
+
+
+def pytest_configure(config: pytest.Config) -> None:
```

Ruff (0.15.15)

[warning] Unused function argument: `config`

(ARG001)